### ISSUE TITLE
[Settings UI] Use System Default Audio Device checkbox behaviour is mildly confusing

### ISSUE THREAD

**Issue by Zyro1331** [Yellow-Dog-Man/Resonite-Issues#1666]
### Describe the issue.

The Use System Default Input / Output Device toggle is confusing. 
I entered the menu to configure what device to use by default, but noticed that the actual Input the game was receiving didn't switch. I was confused and thought it was a bug at first until I realized I need to disable the Use System Default toggle before Resonite will actually utilize that device.
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/49619936/83e2358a-1371-4e89-a652-99ee252a3b80)

This is confusing without any sorta UI feedback, I was using the wrong Input and scratching my head wondering why my microphone wasn't working for an embarrassingly long time.

### To Reproduce

Attempt to change your Audio Input device in-game while the Use System Default Input Device is enabled.

### Expected behavior

It would be much easier to understand that manually configuring the Input Device only works when Use System Default is disabled. 
To portray this limitation to the end user, the button to Manually configure Inputs or Outputs should either stay disabled or be completely hidden when Use System Default is enabled.

Alternatively, you could also make the toggle automatically disable when someone starts manually configuring their devices and setting a different device to their default.

### Resonite Version Number

2024.4.11.484

### What Platforms does this occur on?

Windows

### Log Files

[MANGOVRG - 2024.4.11.484 - 2024-04-11 14_22_05.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/14951554/MANGOVRG.-.2024.4.11.484.-.2024-04-11.14_22_05.log)

### Reporters

@zyro1331

**Comment by Zyro1331** [comment]
> As of the latest build- settings have a description of what they do while hovering over the setting- do you feel this is sufficient to clear up how this setting works, @Zyro1331?
> 
> ![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/39b8ca75-396a-4865-9a96-3136bbaf8ad1)
> 

As the settings descriptions are very useful, you cannot expect all users to be the most attentive towards it when they're just trying to get their microphone working quickly. I have resolved this confusion myself but feel that it could become a bother for others once the Settings UI is pushed to main.

I still feel like the alternate solutions I gave in my report would help with this a lot and reduce the amount of people automatically using the incorrect audio devices in-game, and getting confused about how to manually override it.

This issue comes from a concern that was raised when testing the prerelease on April 4th, when I visited Ryuvi as they worked on content for the Settings UI. Two participants who joined the session were on desktop mode and had no microphone output and weren't able to get it working. I had to clarify how the Use System Default toggle overrides your manually configured inputs, and at the time the description feature didn't exist yet. 

Even with the description feature now, I still feel that the UI should implement that small feedback I mentioned earlier to silently show exactly how the Use System Default toggle behaves to the end user as a small enhancement.
